db/schema.rb

```diff
@@ -1080,6 +1080,15 @@
     t.index ["tag_id"], name: "index_tag_follows_on_tag_id"
   end
 
+  create_table "tag_trends", force: :cascade do |t|
+    t.bigint "tag_id", null: false
+    t.float "score", default: 0.0, null: false
+    t.integer "rank", default: 0, null: false
+    t.boolean "allowed", default: false, null: false
+    t.string "language"
```

Comment:
When I run this migration locally after pulling down latest main (with this in it), I see a diff here:

```
-    t.string "language"
+    t.string "language", default: "", null: false
```

...it seems like based on the migration, that change should have been here?

Am I missing something? Want a PR with the changes? (I also see some annotation diff, but that might be PG version or order-only stuff)

Reply:
Yes please, I’m sorry about the mistake.